src/api.js

```diff
@@ -555,7 +555,24 @@ var API = function(options) {
     assert(/[A-Z][A-Za-z0-9]*/.test(key), 'Invalid error code: ' + key)
     assert(typeof(value) === 'number', 'Expected HTTP status code to be int');
   });
-  this._entries = [];
+  this._entries = [
+    {
+      method:   'get',
+      route:    '/ping',
+      name:     'ping',
+      stability:  'stable',
+      title:    'Ping Server',
+      description: [
+          'Respond without doing anything.',
+          'This endpoint is used to check that the service is up.'
+        ].join('\n'),
+      handler: function(req, res) {
+          res.status(200).json({
+            alive:    true,
+            uptime:   process.uptime()
+            });
+        }
+     }];
```

Comment:
I was planning to eliminate the other ping declarations. I'm not sure though. What do you think?